packages/fxa-content-server/app/tests/spec/views/settings/clients.js

```diff
@@ -4,20 +4,20 @@
 
 'use strict';
 
-const $ = require('jquery');
+import $ from 'jquery';
 const _ = require ('underscore');
```

Comment:
We missed one here! Let's fix that as a follow on.

Reply:
@shane-tomlinson I don't know why grep missed it https://pastebin.com/vceLcK54

Reply:
but I've converted it too.